flixel/addons/editors/tiled/TiledMap.hx

```diff
@@ -31,30 +31,29 @@ class TiledMap
 	
 	public var properties:TiledPropertySet;
 	
+	public var tilesets: Map<String, TiledTileSet>;
```

Comment:
It's with {} it adds the spaces. In a previous version I had a struct in there, and that's when the spaces were added.